### PR TITLE
Add latest Welsh translations for the CCS schema (cy)

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

Updates the CSS schema `.po` file with the latest translations in Welsh (cy) from Crowdin.

### How to review

Check the translations match Crowdin.

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### CODE REVIEW

**Comment by petechd** [comment]
[Download PO file for CCS (xs)](https://trello.com/c/b5lImFqK/4409-download-po-file-for-ccs-xs)

**Review by berroar — APPROVED** [review]

File: translations/cy/ccs_household_gb_wls_cy.po [hunk 1]
```diff
@@ -1,3397 +1,2614 @@
...
 msgstr ""
-"Project-Id-Version: PROJECT VERSION\n"
+"Project-Id-Version: eq-census\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2020-07-22 16:11+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"POT-Creation-Date: 2020-12-07 09:43+0000\n"
+"PO-Revision-Date: 2020-12-16 07:44\n"
+"Last-Translator: \n"
```
Comment (JamesGardiner) [inline translations/cy/ccs_household_gb_wls_cy.po:7]: Should this be populated?

Comment (petechd) [reply translations/cy/ccs_household_gb_wls_cy.po:7]: It's part of our pot files now, so it's what babel 2.9.0 introduced probably..

Comment (petechd) [reply translations/cy/ccs_household_gb_wls_cy.po:7]: This is how it came down, so I'm not sure what value should be there?

**Review by JamesGardiner — APPROVED** [review]

**Comment by JamesGardiner** [comment]
I have approved, but there are outstanding questions about the consistency of the apostrophes used. I've updated Donna, who is going to go back to CrowdIn to check how the inconsistency has appeared.

**Pull request merged**